src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -85,15 +87,16 @@ class TextureUsageTracking extends ValidationTest {
     view: GPUTextureView,
     bindingType: GPUBindingType,
     dimension: GPUTextureViewDimension,
-    bindingTexFormat: GPUTextureFormat | undefined
+    bindingTexFormat: GPUTextureFormat | undefined,
+    compute: boolean
```

Comment:
I thought about this as well when I wrote the tests. We do have tests for visibility, which will test the whole visibility (vertex + fragment + compute). But for other tests that are not intended to test visibility, I think it might be a little better to use the visibility it really depends on.

Reply:
It just requires piping the compute boolean, I think it would be slightly more readable if we didn't have to do it.

Reply:
What do you mean "it just requires piping the compute boolean", sounds like the tiny change you are asking is to pipe the compute boolean. Do you still think that it is slightly better to remove "compute" parameter and always use COMPUTE | FRAGMENT for visibility for all tests? I will do that if you still think so. It's not a big deal, anyway.

Reply:
Readability of code is a lot of small things that add up. If we don't need this boolean, then it's marginally better readability to remove it.

Reply:
Agreed. I hesitated to add 'compute' parameter too: It could make the code more verbose but I thought it might be good to use the visibility the tests really depends on. It's a trade-off but I chose a different one. It's done right now.